service/src/main/java/uk/gov/hmcts/reform/fpl/model/notify/hearing/UpcomingHearingNotifyData.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmcts.reform.fpl.model.notify.hearing;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Data;
+import uk.gov.hmcts.reform.fpl.model.notify.NotifyData;
+
+@Builder
+@Data
+public class UpcomingHearingNotifyData implements NotifyData {
+    @JsonProperty("hearing_date")
```

Comment:
did we know/test if that this serialisation works?

Reply:
added a test in `UpcomingHearingsContentProviderTest`